common/txmgr/types/forwarder_manager.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import (
+	"github.com/smartcontractkit/chainlink/core/services"
+)
+
+type ForwarderManager[ADDR any] interface {
+	services.ServiceCtx
+	// Name change b/c no distinction b/w EOA and contracts for some chains
```

Comment:
Remove this comment about name change. It won't make sense to the reader who didn't see the code before this PR :)

This comment is ok to be added as a PR comment, just not in the code.